ArduCopter/mode.h

```diff
@@ -115,6 +115,14 @@ class Mode {
     // in modes that support landing
     void land_run_horizontal_control();
     void land_run_vertical_control(bool pause_descent = false);
+    void run_land_controllers(bool pause_descent = false) {
+        land_run_horizontal_control();
+        land_run_vertical_control(pause_descent);
+    }
+
+    // returns true if pilot is reposition the vehicle manually in landing modes
```

Comment:
```suggestion
    // returns true if pilot is repositioning the vehicle manually in landing modes
```

... but we proabbly don't need this